tests/src/Functional/ContentEventRenderTest.php

```diff
@@ -130,6 +80,24 @@ public function testEventPage(): void {
       trim($content_banner->filter('.oe-sc-event__oe-summary')->text())
     );
 
+    $date = $crawler->filter('dl > dd');
+
+    // Assert event dates starting and ending same day.
+    $this->assertEquals('Wednesday 09 February 2022, 21.00-23.00 (CET)', trim($date->text()));
+
+    // Assert event dates starting and ending at different days.
```

Comment:
These "Assert" comments feel wrong to me, if they are not on top of an actual assertion.
Here we are just setting things up, the assert happens later. ("arrange, act, assert")